books/Visualization.md

```
@@ -86,6 +86,68 @@ def __init__(self, val=0, next=None):
         self.next = next
 ```
 
+## 1431. Kids With the Greatest Number of Candies [Easy]
+https://leetcode.com/problems/kids-with-the-greatest-number-of-candies/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)